flocker/node/test/test_docker.py

```diff
@@ -170,6 +170,23 @@ def got_list(units):
             d.addCallback(got_list)
             return d
 
+        def test_null_environment(self):
+            """
+            A container that does not include any environment variables
+            contains an empty ``environment`` in the return ``Unit``.
+            """
+            client = fixture(self)
+            name = random_name()
+            self.addCleanup(client.remove, name)
+            d = client.add(name, u"busybox:latest")
+            d.addCallback(lambda _: client.list())
+
+            def got_list(units):
+                unit = [unit for unit in units if unit.name == name][0]
+                self.assertIsNone(unit.environment)
+            d.addCallback(got_list)
+            return d
+
```

Comment:
Doesn't just look identical.... I think it is identical.  It even has the same name.  Accidental duplication, probably?  Just needs to be deleted?